src/box/memtx_engine.c

```diff
@@ -50,10 +50,16 @@
 #include "schema.h"
 #include "gc.h"
 #include "raft.h"
+#include "allocator.h"
 
 /* sync snapshot every 16MB */
 #define SNAP_SYNC_INTERVAL	(1 << 24)
 
+static global_enter_delayed_free_mode memtx_allocator_enter_delayed_free_mode;
+static global_leave_delayed_free_mode memtx_allocator_leave_delayed_free_mode;
+static global_allocator_destroy memtx_allocator_destroy;
+static global_allocator_init memtx_allocator_init;
```

Comment:
Why are these global variables? We have memtx_engine to all memtx-specific things. These clearly are memtx-specific.

Reply:
Fixed